tests/src/main/resources/sparkconf/pod-cohort-evaluator-spark.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+kind: Pod
+apiVersion: v1
+metadata:
+    name: cohort-evaluator-spark
+spec:
+    # The cohort-evaluator-spark Docker image runs the application
+    # under the "spark" user which is created with uid 185
+    securityContext:
+        runAsUser: 185
+        fsGroup: 185
```

Comment:
This file is a duplicate of cohort-evaluator-spark/pod-cohort-evaluator-spark.yaml. It would be nice if we didn't have multiples floating around, especially since these uid/gid values aren't directly sync'ed with the Dockerfile where they are created. I'm not clear on what the best way to do that would be though. It is generally frowned upon to copy files from a separate module.